generify.js

```diff
@@ -55,11 +56,18 @@ function generify (source, dest, data, onFile, done) {
     .on('error', done)
 
   function copyAndReplace (source, dest) {
-    pump(
-      fs.createReadStream(source),
-      split(replaceLine),
-      fs.createWriteStream(dest),
-      complete)
+    if (isBinary(source)) {
+      pump(
+        fs.createReadStream(source),
+        fs.createWriteStream(dest),
+        complete)
+    } else {
+      pump(
+        fs.createReadStream(source),
+        split(replaceLine),
+        fs.createWriteStream(dest),
+        complete)
```

Comment:
A faster and cleaner implementation is to use https://nodejs.org/api/fs.html#fs_fs_copyfile_src_dest_flags_callback instead of streams.

Reply:
`fs.copyFile` seems to be added in v8.5.0, thus it is not suppported by Node 6.